template.yaml

```diff
@@ -57,9 +57,9 @@ Outputs:
   HelloWorldApi:
     Description: "API Gateway endpoint URL for Prod stage for Hello World function"
     Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
-  HelloWorldFunction:
+  HelloWorldFunctionArn:
     Description: "Hello World Lambda Function ARN"
     Value: !GetAtt HelloWorldFunction.Arn
-  HelloWorldFunctionIamRole:
+  HelloWorldFunctionIamRoleArn:
```

Comment:
Looks like you renamed the function role to be independent of function, could it not be extracted (along with the logsPolicy) to another template and imported in this one (that would contain the API)?

Reply:
We can entirely remove the LogsPolicy in favor of `arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole` which is identical

Reply:
@xschildw maybe? it would be a good idea if it's possible.